gpflow/conditionals.py

```diff
@@ -22,7 +22,7 @@
 from .features import Kuu, Kuf, InducingPoints, InducingFeature
 from .kernels import Kernel, Combination
 from .probability_distributions import Gaussian
-
+from .misc import tile_over_new_axis, swap_final_dims
```

Comment:
Proposal to rename functions:
* `tile_over_new_axis` => `repeat`
* `swap_final_dims` => `transpose`, so that if people hasn't specified axes function permutes last two dims.

Reply:
happy to make these changes, but doesn't that overload the meanings, as tensorflow already implements `repeat` and `transpose`, which have different behaviour?

Reply:
annoyingly the behaviour of the tensorflow functions isn't really consistent with the rest of the broadcasting conventions, but there you go...